### PR TITLE
Docs harmonia

### PR DESCRIPTION
**Pull request by rpelisse**

### CODE REVIEW

File: job-dsl-lib/eap7/Builder.groovy [hunk 1]
```diff
@@ -81,14 +81,17 @@ class Builder {
...
             }
             stringParam {
                 name("HARMONIA_BRANCH")
                 defaultValue(harmoniaBranch)
+                description("The is the name of the branch of the Harmonia folder to use for this job. Normally it's should be 'main', but it can be changed to experiment changes on the Harmonia's script without affecting other jobs.")
             }
             stringParam {
                 name("HARMONIA_REPO")
                 defaultValue("https://github.com/jboss-set/harmonia.git")
+                description("This the URL to the Harmonia's repo used. It can be override so that user can create their own fork, implement anychange they want, without having to modify the pipeline directly.")
```
Comment (iweiss) [inline job-dsl-lib/eap7/Builder.groovy:94]: anychange => any change

File: job-dsl-lib/eap7/Builder.groovy [hunk 2]
```diff
@@ -81,14 +81,17 @@ class Builder {
             stringParam {
                 name ("HARMONIA_SCRIPT")
                 defaultValue(harmoniaScript)
+                description("The is variable is the path, inside the Harmonia folder to the script being used. The job will execute the script or report an error if the script does not exist inside the folder.")
             }
             stringParam {
                 name("HARMONIA_BRANCH")
                 defaultValue(harmoniaBranch)
+                description("The is the name of the branch of the Harmonia folder to use for this job. Normally it's should be 'main', but it can be changed to experiment changes on the Harmonia's script without affecting other jobs.")
```
Comment (iweiss) [inline job-dsl-lib/eap7/Builder.groovy:89]: The is => This is

File: job-dsl-lib/eap7/Builder.groovy [hunk 3]
```diff
@@ -81,14 +81,17 @@ class Builder {
             stringParam {
                 name ("HARMONIA_SCRIPT")
                 defaultValue(harmoniaScript)
+                description("The is variable is the path, inside the Harmonia folder to the script being used. The job will execute the script or report an error if the script does not exist inside the folder.")
```
Comment (iweiss) [inline job-dsl-lib/eap7/Builder.groovy:84]: The is => This is

File: pipelines/ansible-pipeline [hunk 4]
```diff
@@ -137,7 +137,7 @@ pipeline {
         success {
             script {
               // The release job will take care of trigger jobs in the right order
-              if ( "${env.RELEASE_NAME}" == "null" && ! "zeus".equals("${env.PROJECT_NAME}") ) {
+              if ( ("${env.RELEASE_NAME}" == "null" || "${env.RELEASE_NAME}" == "TEST") && ! "zeus".equals("${env.PROJECT_NAME}") ) {
```
Comment (iweiss) [inline pipelines/ansible-pipeline:140]: String "null"?

File: job-dsl-lib/eap7/Builder.groovy [hunk 5]
```diff
@@ -81,14 +81,17 @@ class Builder {
...
             }
             stringParam {
                 name("HARMONIA_BRANCH")
                 defaultValue(harmoniaBranch)
+                description("The variable contains the name of the branch of the Harmonia folder to use for this job. Normally it's should be 'main', but it can be changed to experiment changes on the Harmonia's script without affecting other jobs.")
             }
             stringParam {
                 name("HARMONIA_REPO")
                 defaultValue("https://github.com/jboss-set/harmonia.git")
+                description("This the URL to the Harmonia's repo used. It can be override so that user can create their own fork, implement anychange they want, without having to modify the pipeline directly.")
```
Comment (iweiss) [inline job-dsl-lib/eap7/Builder.groovy:94]: anychange => any change

**Review by spyrkob — APPROVED** [review]

**Pull request merged**